ui/component/transactionListTable/view.jsx

```diff
@@ -45,15 +40,24 @@ function TransactionListTable(props: Props) {
               </tr>
             </thead>
             <tbody>
-              {transactionList.map(t => (
-                <TransactionListItem
-                  key={`${t.txid}:${t.nout}`}
-                  transaction={t}
-                  reward={rewards && rewards[t.txid]}
-                  isRevokeable={isRevokeable(t.txid, t.nout)}
-                  revokeClaim={revokeClaim}
-                />
-              ))}
+              {transactions &&
```

Comment:
we're moving to two entirely separate ways of rendering transactions? this seems like a step backwards - why is this necessary?

Reply:
deleted